15-morning-sun-star/api_definitions/rest/user_registry.yaml

```diff
@@ -0,0 +1,257 @@
+          schema:
+            title: Internal ID
+            type: string
+          required: true
+        - in: query
+          name: issuer
+          description: Issuer if an external ID is specified
+          schema:
+            title: Issuer
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/User"
+          description: Successful Response
+        "401":
+          description: Not authorized to get user data.
+        "404":
+          description: The user was not found.
+        "422":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/HTTPValidationError"
+          description: Validation error in submitted user identification.
+
+    patch:
+      operationId: patch_user
+      summary: Modify user data
+      description: Endpoint used to modify the user data for a specified user
+      tags:
+        - users
+      parameters:
+        - in: path
+          name: id
+          description: Internal ID
+          schema:
+            title: Internal ID
+            type: string
+          required: true
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                status:
+                  $ref: "#/components/schemas/FullName"
+                test_result:
+                  $ref: "#/components/schemas/EMail"
+                test_date:
+                  $ref: "#/components/schemas/UserStatus"
+        required: true
+      responses:
+        "204":
+          description: User data was successfully saved.
+        "404":
+          description: The user was not found.
+        "422":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/HTTPValidationError"
+          description: Validation error in submitted user data.
+
+    delete:
+      operationId: delete_user
+      summary: Delete user
+      description: Endpoint used to get the user data for a specified user
+      tags:
+        - users
+      parameters:
+        - in: path
+          name: id
+          description: Internal ID
+          schema:
+            title: Internal ID
+            type: string
+          required: true
+      responses:
+        "204":
+          description: User was successfully deleted.
+        "422":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/HTTPValidationError"
+          description: Validation error in submitted user identification.
+
+components:
+
+  schemas:
+  
+    User:
+      type: object
+      title: User
+      properties:
+        id:
+          title: Internal ID
+          type: string
+        ls_id:
+          title: LS AAI ID
+          type: string
```

Comment:
maybe store as issuer/external ID pair like in the request body.

Reply:
Using issuer and subject now, see above.